superset-frontend/plugins/plugin-chart-echarts/test/Timeseries/transformProps.test.ts

```diff
@@ -132,37 +137,40 @@ describe('EchartsTimeseries tranformProps', () => {
 
   it('should add an interval, event and timeseries annotation to viz', () => {
     const event: EventAnnotationLayer = {
-      annotationType: 'EVENT',
+      annotationType: AnnotationType.Event,
       name: 'My Event',
       show: true,
-      sourceType: 'NATIVE',
-      style: 'solid',
+      showLabel: true,
+      sourceType: AnnotationSourceType.Native,
+      style: AnnotationStyle.Solid,
       value: 1,
     };
 
     const interval: IntervalAnnotationLayer = {
-      annotationType: 'INTERVAL',
+      annotationType: AnnotationType.Interval,
       name: 'My Interval',
       show: true,
-      sourceType: 'table',
+      showLabel: true,
+      sourceType: AnnotationSourceType.Table,
       titleColumn: '',
       timeColumn: 'start',
       intervalEndColumn: '',
       descriptionColumns: [],
-      style: 'dashed',
+      style: AnnotationStyle.Dashed,
       value: 2,
     };
 
     const timeseries: TimeseriesAnnotationLayer = {
-      annotationType: 'TIME_SERIES',
+      annotationType: AnnotationType.Timeseries,
       name: 'My Timeseries',
       show: true,
-      sourceType: 'line',
-      style: 'solid',
+      showLabel: true,
+      sourceType: AnnotationSourceType.Line,
+      style: AnnotationStyle.Solid,
       titleColumn: '',
       value: 3,
     };
-    const chartProps = new ChartProps({
+    const chartProps: unknown = new ChartProps({
```

Comment:
Just marking this with the other unknowns, in case this can be described with [one of these prop types](https://github.com/apache/superset/blob/6f8927209b80dc61ba14fc66ff94871b36a83321/superset-frontend/plugins/plugin-chart-echarts/src/Timeseries/transformProps.ts#L32-L37) instead of `unknown`.